crates/swc/tests/fixture/issue-2669/3/input/index.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace TEST {
+  export const VALUE = "value";
+}
+export default TEST;
```

Comment:
Actually typescript would even emit export for this case as below:

cf. https://www.typescriptlang.org/play?target=1&module=1#code/CYUwxgNghgTiAEA7KBbEBnADlMCAqAogMp7wDeAUPPCAB6YD2MALvGA4uqwGoCCAMgFUC8ALzwARADcoEAK4gJAbgoBfCnUYt4oAGZQ5EVoRJKgA

```js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.default = TEST;
```

So, it might be fine to actually allow emitting any namespaces.
Please let me know if that's preferable.

Reply:
I think it's better to follow the behavior of `tsc`.

Reply:
Thanks for the comment!
I'm adding more tests I can think of and I will try to align with what tsc does.